package.json

```diff
@@ -63,26 +63,27 @@
     "npm": ">= 6.0.0"
   },
   "dependencies": {
-    "@energyweb/credential-governance": "2.2.1-alpha.316.0",
+    "@energyweb/credential-governance": "^2.2.1-alpha.318.0",
     "@energyweb/ekc": "^0.6.7",
-    "@energyweb/onchain-claims": "2.2.1-alpha.316.0",
+    "@energyweb/onchain-claims": "^2.2.1-alpha.318.0",
     "@energyweb/staking-pool": "^1.0.0-rc.14",
-    "@energyweb/vc-verification": "2.2.1-alpha.316.0",
+    "@energyweb/vc-verification": "^2.2.1-alpha.318.0",
     "@ensdomains/ens": "^0.6.2",
-    "@ew-did-registry/claims": "0.8.1-alpha.1037.0",
-    "@ew-did-registry/credentials-interface": "0.8.1-alpha.1037.0",
-    "@ew-did-registry/did": "0.8.1-alpha.1037.0",
-    "@ew-did-registry/did-document": "0.8.1-alpha.1037.0",
-    "@ew-did-registry/did-ethr-resolver": "0.8.1-alpha.1037.0",
-    "@ew-did-registry/did-ipfs-store": "0.7.1-alpha.816.0",
-    "@ew-did-registry/did-resolver-interface": "0.8.1-alpha.1037.0",
-    "@ew-did-registry/jwt": "0.8.1-alpha.1037.0",
-    "@ew-did-registry/keys": "0.8.1-alpha.1037.0",
-    "@ew-did-registry/proxyidentity": "0.8.1-alpha.1037.0",
-    "@ew-did-registry/revocation": "0.8.1-alpha.1037.0",
+    "@ew-did-registry/claims": "^0.8.1-alpha.1122.0",
+    "@ew-did-registry/credentials-interface": "^0.8.1-alpha.1122.0",
+    "@ew-did-registry/did": "^0.8.1-alpha.1122.0",
+    "@ew-did-registry/did-document": "^0.8.1-alpha.1122.0",
+    "@ew-did-registry/did-ethr-resolver": "^0.8.1-alpha.1122.0",
+    "@ew-did-registry/did-ipfs-store": "^0.8.1-alpha.1122.0",
```

Comment:
I'm not sure we want to update `did-ipfs-store` yet, until we are confident that EW's IPFS cluster infrastructure is suitable. Let's discuss this further